src/iso_alloc.c

```diff
@@ -401,6 +393,9 @@ INTERNAL_HIDDEN iso_alloc_zone_t *_iso_new_zone(size_t size, bool internal, int3
         return NULL;
     }
 
+    /* This is a good time to refresh the root rng seed */
```

Comment:
Why is it a good time to do so?

Reply:
This code path is invoked every time we make a new zone or retire/refresh an existing one. Because we utilize the prng in canary and freelist creation (which both happen as a result of this function) it makes sense to reset the state of wyrand from a safe random source. I can fix this comment.